gramex/apps/admin2/index.html

```diff
@@ -187,12 +187,8 @@ <h4 class="alert-heading">User management is not enabled</h4>
       <div class="h1 text-danger">Work in progress</div>
 
     {% elif current_tab == 'nlg' %}
-      {% set action = handler.get_argument('action', '') %}
-      {% if action == 'edit' %}
-        {% include "../nlg/index.html" %}
-      {% else %}
-        {% include "../nlg/grmform.html" %}
-      {% end %}
+      {% set nlg_base = 'nlg/' %}
+      {% include "../nlg/index.html" %}
```

Comment:
`nlg/index.html` is a full-bodied HTML file. Can't be `{% include %}`-ed here like this. Put the central `div` of the page in a separate template and include that here. Modify `nlg/index.html` to include that dive too.